### DISCUSSION TITLE
右键收藏按钮收藏到默认收藏夹

### DISCUSSION THREAD

**Discussion by Chronodeus** [ywmoyue/biliuwp-lite#787] [General]
### 问题描述

很多时候收藏视频只需要默认，而此时还需要点开目录勾选后再点确定，相对而言较为麻烦

### 描述解决方案

跟点赞按钮右键三连一样，给收藏按钮加个右键收藏到默认收藏夹即可

### 代码贡献

否

**Comment by ywmoyue** [comment]
4.7.1已默认启用快速收藏，试试是否满足需求 @Chronodeus
